docs/operations/containerized_data_importer.md

```diff
@@ -101,16 +101,140 @@ uploading the file to the `cdi-uploadproxy`.
+
+    Post https://192.168.39.32:31001/v1alpha1/upload: x509: cannot validate certificate for 192.168.39.32 because it doesn't contain any IP SANs
+
+
+It is easily fixed by adding an entry it your local name resolution service.
+This could be a DNS server or the local hosts file.
+The URL used to upload the proxy should be changed to reflect the resolvable name.
+
+The `Subject` and the `Subject Alternative Name` in the certificate contain valid names that can be used for resolution.
+Only one of these names needs to be resolvable.
+Use the `openssl` command to view the names of the cdi-uploadproxy service.
+
+    echo | openssl s_client -showcerts -connect 192.168.39.32:31001 2>/dev/null \
+         | openssl x509 -inform pem -noout -text \
+         | sed -n -e '/Subject.*CN/p' -e '/Subject Alternative/{N;p}'
+
+        Subject: CN = cdi-uploadproxy
+            X509v3 Subject Alternative Name: 
+                DNS:cdi-uploadproxy, DNS:cdi-uploadproxy.cdi, DNS:cdi-uploadproxy.cdi.svc
+
+Adding the following entry to the /etc/hosts file, if it provides name resolution, should fix this issue.
+Any service that provides name resolution for the system could be used.
+
+    cat "192.168.39.32  mycluster.example.org" >> /etc/hosts
+
+The upload should now work.
+
+    virtctl image-upload dv f33 \
+      --size 5Gi \
+      --image-path Fedora-Cloud-Base-33-1.2.x86_64.raw.xz \
+      --uploadproxy-url https://cdi-uploadproxy:31001
+
+    PVC default/f33 not found 
+    DataVolume default/f33 created
+    Waiting for PVC f33 upload pod to be ready...
+    Pod now ready
+    Uploading data to https://cdi-uploadproxy:31001
+
+     193.89 MiB / 193.89 MiB [=============================================] 100.00% 1m38s
+
+    Uploading data completed successfully, waiting for processing to complete, you can hit ctrl-c without interrupting the progress
+    Processing completed successfully
+    Uploading Fedora-Cloud-Base-33-1.2.x86_64.raw.xz completed successfully
+
+
+
+#### Certificate Signed by Unknown Authority
+This happens because the cdi-uploadproxy certificate is self signed and the system does not trust the cdi-uploadproxy as a Certificate Authority.
+
+    virtctl image-upload dv f33 \
+      --size 5Gi \
+      --image-path Fedora-Cloud-Base-33-1.2.x86_64.raw.xz \
+      --uploadproxy-url https://cdi-uploadproxy:31001
+
+    PVC default/f33 not found 
+    DataVolume default/f33 created
+    Waiting for PVC f33 upload pod to be ready...
+    Pod now ready
+    Uploading data to https://cdi-uploadproxy:31001
+
+     0 B / 193.89 MiB [-------------------------------------------------------]   0.00% 0s
+
+    Post https://cdi-uploadproxy:31001/v1alpha1/upload: x509: certificate signed by unknown authority
+
+This can be fixed by adding the certificate to the systems trust store.
+Download the cdi-uploadproxy-server-cert.
+
+    oc get secret -n cdi cdi-uploadproxy-server-cert \
```

Comment:
`oc` is a vendor specific command.  We can't use it in KubeVirt context.  Please change to `kubectl`

Reply:
sorry, cant believe I did that. I just fixed it to kubectl, even verified it worked.